app/models/concerns/dossier_rebase_concern.rb

```diff
@@ -72,11 +71,9 @@ def rebase
 
     # due to repetition tdc clone on update or erase
     # we must reassign tdc to the latest version
-    Champ
-      .includes(:type_de_champ)
-      .where(dossier: self)
-      .map { [_1, target_coordinates_by_stable_id[_1.stable_id].type_de_champ] }
-      .each { |champ, target_tdc| champ.update_columns(type_de_champ_id: target_tdc.id, rebased_at: Time.zone.now) }
+    champs
+      .map { [_1, target_coordinates_by_stable_id[_1.stable_id].type_de_champ_id] }
+      .each { |champ, type_de_champ_id| champ.update_columns(type_de_champ_id:) }
```

Comment:
pourquoi pas directement faire ?

```suggestion
      .each { _1.update_columns(type_de_champ_id:  target_coordinates_by_stable_id[_1.stable_id].type_de_champ_id]) }
```

 (je reconnais qu'on s'en fout un peu, à part que sur une grosse démarche ça peut aller plus vite)